### PR TITLE
feat: add css color picker

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
Add a css color picker into tooling

image

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-tooling-react/src/css-editor/color/color.style.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import {
+    applyControlRegion,
+    applyControlWrapper,
+    applyInputStyle,
+    applyLabelStyle,
+} from "../../style";
+
+export interface CSSColorClassNameContract {
+    cssColor?: string;
+    cssColor_colorInputContainer?: string;
```
Comment (janechu) [inline packages/fast-tooling-react/src/css-editor/color/color.style.ts]: can you change this to `cssColor_colorInputRegion`?

File: packages/fast-tooling-react/src/css-editor/editor.tsx [hunk 2]
```diff
@@ -9,6 +9,8 @@ import { CSSPosition, CSSPositionValues, Location, PositionValue } from "./posit
 import { CSSSpacing, CSSSpacingValues, SpacingProperty } from "./spacing";
 import { CSSWidth, CSSWidthValues } from "./width";
 import { CSSHeight, CSSHeightValues } from "./height";
+import { CSSColor, CSSColorValues } from "./color";
+import { CSSBorder, CSSBorderValues } from "./border";
```
Comment (janechu) [inline packages/fast-tooling-react/src/css-editor/editor.tsx]: did this come from another PR? I don't see the border files here.

Comment (eljefe223) [reply packages/fast-tooling-react/src/css-editor/editor.tsx]: good catch yeah I built them together and then separated them out into 2 PRs this must have been left over

File: packages/fast-tooling-react/src/css-editor/color/index.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import manageJss from "@microsoft/fast-jss-manager-react";
```
Comment (nicholasrice) [inline packages/fast-tooling-react/src/css-editor/color/index.ts]: You need to import react here - I'm getting  a build error cause it's missing.

File: packages/fast-tooling-react/src/css-editor/color/color.props.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+export interface CSSColorValues {
+    color?: string;
+}
+
+export interface CSSColorUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+export interface CSSColorHandledProps
+    extends ManagedClasses<CSSColorClassNameContract> {
+    /**
+     * The data
```
Comment (nicholasrice) [inline packages/fast-tooling-react/src/css-editor/color/color.props.ts]: Whats the purpose for making this an object type? Should this simply accept "color"?

Comment (eljefe223) [reply packages/fast-tooling-react/src/css-editor/color/color.props.ts]: we did that elsewhere (ie height).

Comment (janechu) [reply packages/fast-tooling-react/src/css-editor/color/color.props.ts]: for clarity, any one of these components could be editing one or multiple CSS properties, so we are keeping the same API even though there may only be a single property being updated.

**Pull request merged**